src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,95 @@
 
 namespace System.Text.Json
 {
+    public partial class JsonArray : System.Text.Json.JsonNode, System.Collections.Generic.ICollection<System.Text.Json.JsonNode>, System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode>, System.Collections.Generic.IList<System.Text.Json.JsonNode>, System.Collections.IEnumerable
+    {
+        public JsonArray() { }
+        public JsonArray(System.Collections.Generic.IEnumerable<bool> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<byte> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<double> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<short> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<int> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<long> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<sbyte> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<float> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<string> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<ushort> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<uint> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<ulong> jsonValues) { }
+        public int Count { get { throw null; } }
+        public bool IsReadOnly { get { throw null; } }
+        public System.Text.Json.JsonNode this[int idx] { get { throw null; } set { } }
+        public void Add(bool jsonValue) { }
+        public void Add(byte jsonValue) { }
+        public void Add(double jsonValue) { }
+        public void Add(short jsonValue) { }
+        public void Add(int jsonValue) { }
+        public void Add(long jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(sbyte jsonValue) { }
+        public void Add(float jsonValue) { }
+        public void Add(string jsonValue) { }
+        public void Add(System.Text.Json.JsonNode jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ushort jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(uint jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ulong jsonValue) { }
+        public void Clear() { }
+        public bool Contains(bool jsonValue) { throw null; }
```

Comment:
Can a JsonArray contain duplicate values? I assume so, in that case what would we return here? true if it is found at least once I guess?

Reply:
You are right - `JsonArray` can contain duplicate values. Returning `true` if at least one value was found seems reasonable.